tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -2797,6 +2797,17 @@ fn write_example_wire_data_to_file() {
     f.write_all(b.finished_data()).unwrap();
 }
 
+#[test]
+fn binary_data_from_future_source() {
+    let bytes = load_file("../monsterdata_unknown.mon").expect("missing monsterdata_unknown.mon");
+    let correct = my_game::example::monster_buffer_has_identifier(&bytes);
+    assert!(correct);
+
+    let m = my_game::example::get_root_as_monster(&bytes);
+    assert_eq!(m.name(), "FromFuture");
+    assert!(m.signed_enum().is_none());
+}
+
```

Comment:
@jean-airoldie Is this sufficient for testing of Option<None> enum?

Reply:
Yeah that's good stuff.